addons/account/models/account_move.py

```diff
@@ -114,6 +114,52 @@ def _onchange_date(self):
         help="Technical field used to keep track of the tax cash basis reconciliation."
         "This is needed when cancelling the source: it will post the inverse journal entry to cancel that part too.")
 
+    @api.onchange('line_ids')
+    def _onchange_line_ids(self):
+        '''Compute additional lines corresponding to the taxes set on the line_ids.
+
+        For example, add a line with 1000 debit and 15% tax, this onchange will add a new
+        line with 150 debit.
+        '''
+        # Retrieve the existing tax lines and drop them.
+        tax_line_ids = self.line_ids.filtered(lambda l: l.is_tax_line)
```

Comment:
why???

Reply:
I don't remember well but it was the only way I found to avoid some problems when dealing with real ids on db + 'fake ids' corresponding to newly created (but not committed) lines... to recheck!